lib/core/lib/ops/utils.js

```diff
@@ -13,36 +17,40 @@ function isUniqueViolationError (err) {
   return err && err.code === '23505'
 }
 
+function isForeignKeyViolationError (err) {
+  return err && err.code === '23503'
+}
+
 function checkPoliciesOrg (db, policies, organizationId, cb) {
-  db.query(SQL`SELECT id FROM policies WHERE id = ANY (${policies}) AND org_id != ${organizationId}`, (err, result) => {
+  db.query(SQL`SELECT id FROM policies WHERE id = ANY (${policies}) AND org_id = ${organizationId}`, (err, result) => {
     if (err) return cb(Boom.badImplementation(err))
 
-    if (result.rowCount !== 0) {
-      return cb(Boom.badRequest(`Some policies [${result.rows.map(r => r.id).join(',')}] were not found`))
+    if (result.rowCount !== policies.length) {
+      return cb(Boom.badRequest(`Some policies [${policies.filter(p => result.rows.map(r => r.id).indexOf(p) < 0)}] were not found`))
```

Comment:
My only worry is if we have big arrays. We are basically doing 2 forEach loops :/ (policies and then for each policy all result.rows)

Maybe using [`_.difference`](https://lodash.com/docs/4.17.4#difference) can help, can you give it a try?